pkg/scheduler/metrics/metrics.go

```diff
@@ -41,7 +41,15 @@ const (
 
 // All the histogram based metrics have 1ms as size for the smallest bucket.
 var (
-	SchedulingLatency = prometheus.NewSummaryVec(
+	scheduleAttempts = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "schedule_attempts_total",
+			Help:      "Number of attempts to schedule pods, by whether or not we succeeded",
+		}, []string{"result"})
+	PodScheduleSuccesses = scheduleAttempts.With(prometheus.Labels{"result": "success"})
+	PodScheduleFailures  = scheduleAttempts.With(prometheus.Labels{"result": "unschedulable"})
```

Comment:
Why is it is `unschedulable ` instead of `failure`?

Reply:
I don't think that `unschedulable` is a failure result. In fact, I am not correctly propagating actual errors to this metric. How about using these cells for the metric:
- ok_scheduled
- ok_unschedulable
- error

Reply:
Or, drop "ok_" and just keep scheduled, unschedulable, and error.